docs/standaard/documenten/consultatie-v1_5_0.md

```diff
@@ -0,0 +1,223 @@
+- bij de betekenis van de laatste twee waarden (`ter_vaststelling` en `vastgesteld`) is het besluitvormingsproces als uitgangspunt genomen, maar ook geprobeerd (met name bij `vastgesteld`) geprobeerd te beschrijven wat het doorlopen van dit proces betekent voor de inhoud van het informatieobject ("het is bekrachtigd").
+
+De Bewerkings- en vaststellingsstatus `definitief` moet niet verward worden met de Archiefstatus `onmuteerbaar`. Die laatste betekent in feite "geen enkele wijziging toegestaan". Voor zover het gaat om door de gemeente gecreëerde informatieobjecten betekent `definitief` daarentegen dat naar aanleiding van overleg, reviews en afspraken een 'overeengekomen' of 'afgestemde' versie van informatieobjectinhoud is ontstaan. Wijzigingen hierin zijn niet verboden, maar zullen in veel gevallen vragen om verantwoording bij doorgevoerde wijzigingen op basis van nieuwe reviews en afspraken.
+
+Rationale en tabel illustreren dat, door voor te stellen bovenstaande wijzigingen in een minor-versie van de Documenten API te verwerken, een situatie ontstaat waarin sommige waarden meerdere betekenissen kunnen hebben. Onder aandachtspunten wordt uitgelegd waarom hiervoor toch is gekozen.
+
+### Gedragsregel 'Zetten Zaak.archiefstatus (zrc-022)'
+
+Link huidige: [Zrc-022](../zaken/index.md/#archiveren)
+
+De standaardwaarde voor archiefstatus [van de zaak, IH] is `nog_te_archiveren`. Indien een andere waarde gezet wordt, dan MOETEN alle gerelateerde informatieobjecten de status `gearchiveerd` (depricated) EN/OF de archiefstatus `onveranderlijk` of `duurzaam_toegankelijk` hebben (zie [archiefstatus](#property-archiefstatus) hieronder).
+
+De attributen Zaak.archiefnominatie en Zaak.archiefactiedatum MOETEN een waarde krijgen als de de archiefstatus een waarde krijgt anders dan nog_te_archiveren.
+
+Indien deze voorwaarden niet voldaan zijn, dan MOET het ZRC met een HTTP 400 foutbericht antwoorden.
+
+#### Rationale voor wijziging
+
+Dat informatieobjecten na afsluiten van een zaak niet meer ('zomaar') gewijzigd kunnen worden, is een algemeen geldende eis. Bovenstaande is daarom een uitvoerbare gedragsregel met algemene geldigheid. De wijziging in deze regel ontstaat omdat statussen die te maken hebben met archivering zijn verhuisd van `status` naar `archiefstatus`. Zowel de archiefstatussen `onveranderlijk` of `duurzaam_toegankelijk` geven invulling aan de eis van onmuteerbaarheid. En zolang de bestaande statuswaarden met de toevoeging 'depricated' gebruikt mogen worden blijft ook registratie van `gearchiveerd` bij `status` een geldige manier om aan deze gedragsregel te voldoen. 
+
+### Gedragsregel 'Statuswijzigingen van informatieobjecten (drc-005)'
+
+Link huidige: [Zrc-022](./index.md/#statuswijzigingen-van-informatieobjecten-drc-005)
```

Comment:
De naam van de link moet [Drc-005] zijn in plaats van [Zrc-022]. De link zelf is gelukkig wel goed :-)

Reply:
Ik heb de wijzigingen in de gedragsregels zrc-022 en drc-005 nu ook meegenomen in mijn eigen PR #2403. Dit heeft tot gevolgd dat naast de drc zrc ook een nieuwe versie van de zrc uitgebracht zal moeten worden.